tools/DARIAH-DE-Geo-Browser-CSV.json

```diff
@@ -0,0 +1,34 @@
+{
+    "task": "Visualisation",
+    "deployment": "development",
```

Comment:
Is the intention to keep it hidden in production?

Reply:
Thanks for the check André, these should be now solved.